packages/create-svelte/shared/+eslint/.eslintignore

```diff
@@ -1,13 +1,7 @@
-.DS_Store
-node_modules
-/build
-/.svelte-kit
 /package
-.env
-.env.*
-!.env.example
+node_modules
 
-# Ignore files for PNPM, NPM and YARN
```

Comment:
I saw the comment that these were removed because you need a yaml/json plugin before these would be formatted (https://github.com/sveltejs/kit/pull/11881#discussion_r1501607538). But I wonder when you install such a plugin, wouldn't you want the lockfiles here? It seems like maybe it could save a step to have them already listed?